.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,7 @@
+[Please delete this text and describe the bug here. Ensure that you are attaching a [Minimal Workable Example](https://stackoverflow.com/help/minimal-reproducible-example), loading the least possible packages. Please also mention the Julia version, the version of the loaded packages (should ideally be the latest), and the complete stack trace.]
+
+Bug report checklist
```

Comment:
I'd drop the checklist. This makes more sense for a PR, where items are often ticked after initially opening. 

Re packages, "the version of the loaded packages" sounds like we want people to transcribe these, and manually check they are latest, which IMO we mostly don't. What I think that should say is that you should run your MWE in a fresh session, in a fresh environment, so that you know it's everything-latest. Maybe explaining `] activate --temp`?